docs/userguide/campaigns.md

```diff
@@ -1,25 +1,146 @@
+
+### Basics
+
+```{attention}
+Obtaining recommendations resp. adding measurements using the `recommend` resp.
+`add_measurements` functions is the only safe way to inform a `Campaign` object about
+new measurements. These functions update the necessary metadata that is crucial for the
+proper execution of a campaign. It is important to rely on these functions to maintain
+the integrity and reliability of the campaign's execution.
+```
+
+To obtain a recommendation for the next experiment, we can query the campaign and use
+the [`recommend`](baybe.campaign.Campaign.recommend) function. The function takes only
+the `batch_quantity` keyword, specifying the desired size of
+the batch of experiments to be conducted.
+
+~~~python
+rec = campaign.recommend(batch_quantity=3)
+~~~
+
+The `recommend` function returns a `DataFrame` with `batch_quantity` many rows, each 
+representing a set of parameters from the search space.
+
+```{important}
+There is a difference between performing multiple recommendations
+with batch size of 1 and a single recommendation with a larger batch size.
+* **Batch size larger than 1**: The recommended experiments
+are chosen to *jointly* optimized the acquisition function.
+This means that the recommendations are made considering the interaction of multiple
+experiments together.
+* **Batch size of 1**: When making several smaller recommendations instead, each
+*individual* recommendation optimizes the acquisition function at the specific point in 
+time when it is requested. In this case, the recommendations are made independently of
+each other without considering the joint optimization.
+
+Note that this distinction might not be relevant or applicable for all possible
+situation, for example when using recommenders that do not perform joint optimization.
+Currently, the sequential greedy recommender is the only available recommender
+performing joint optimization.
+```
+
+### Caching of recommendations
+
+Whenever recommendations are made, the `Campaign` object caches them. If any new 
+measurements are added, then the cached recommendations are deleted. However,
+if no measurements are added and the `recommend` function is called again, then the
+`Campaign` object returns the cached recommendations instead of generating new
+ones. In addition, the cache is also reset if the batch size of the repeated call has
+changed compared to the first one. This is due to the way the batch size influences
+which points are being recommended.
+This caching mechanism helps to optimize performance by avoiding unnecessary
+re-computations when measurements are not provided.
+
+(AM)=
+## Adding measurements
+
+Measurements are added by expanding the  `DataFrame` that was created by the `recommend`
+function by adding a new column for the target. 
+
+~~~python
+rec["Target_max"] = [2, 4, 9]  # 3 values matching the batch_quantity of 3
+campaign.add_measurements(rec)
+new_rec = campaign.recommend(batch_quantity=5)
+~~~
```

Comment:
Again would like to see the corresponding table here, now with the target values. This will make it really easy to visually connect the recommend and add_measurement steps in the guide

Reply:
See my comment above: Not happy to change this since it either requires a full example or some "random dataframe"

Reply:
I am resolving this conversation since this is basically a duplicate of the other discussion.